timify-api/models/podcast.js

```diff
@@ -1,25 +1,55 @@
 import { DataTypes } from 'sequelize';
 import { sequelize } from '../database.js';
+import { ItunesInfo } from './itunesInfo.js';
 
 export const Podcast = sequelize.define('Podcast', {
-  title: {
+  uuid: {
+    type: DataTypes.UUID,
+    defaultValue: DataTypes.UUIDV4,
+    primaryKey: true
+  },
+  name: {
+    type: DataTypes.STRING,
+    allowNull: false
+  },
+  imageUrl: {
     type: DataTypes.STRING,
     allowNull: false
   },
   description: {
     type: DataTypes.TEXT,
     allowNull: false
+},
+  authorName: {
+    type: DataTypes.STRING,
+    allowNull: false
   },
-  coverImage: {
+  datePublished: {
     type: DataTypes.STRING,
     allowNull: false
   },
-  category: {
-    type: DataTypes.TEXT,
+  itunesId: {
+    type: DataTypes.STRING,
     allowNull: false
-},
-  host: {
+  },
+  genre: {
     type: DataTypes.STRING,
     allowNull: false
-  }
+  },
+  rssURL: {
+    type: DataTypes.STRING,
+    allowNull: false
+  },
+  rssOwnerName: {
+    type: DataTypes.STRING,
+    allowNull: false
+  },
+  rssOwnerPublicEmail: {
```

Comment:
i see we're changing the podcast data model to match the output of the api. do we need _all_ of these fields though?